src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9510,17 +9553,22 @@ internal BracketedArgumentListSyntax ParseBracketedArgumentList()
                         // additional arguments
                         while (true)
                         {
-                            if (this.CurrentToken.Kind == SyntaxKind.CloseParenToken ||
-                                this.CurrentToken.Kind == SyntaxKind.CloseBracketToken ||
-                                this.CurrentToken.Kind == SyntaxKind.SemicolonToken)
+                            if (AtEndOfArgumentList())
                             {
                                 break;
                             }
-                            else if (this.CurrentToken.Kind == SyntaxKind.CommaToken || this.IsPossibleArgumentExpression())
+                            else if (this.CurrentToken.Kind == SyntaxKind.CommaToken ||
+                                     this.IsPossibleArgumentExpression())
```

Comment:
This is code style change. Wouldn't it be more appropriate to move that to separate commit?